lib/docs/src/tag.stories.tsx

```diff
@@ -1,61 +1,56 @@
 import { Meta } from "@storybook/react";
-import { Fragment } from "react";
+import { Fragment, useState } from "react";
 import { Button, DivPx, Tag } from "../../core/src";
 import { Utils } from "./utils";
-
-const colors: string[] = [
-	"red",
-	"yellow",
-	"green",
-	"blue",
-	"indigo",
-	"purple",
-	"pink",
-	"gray",
-];
+import { GalleryFeedbackTag } from "../../gallery/src/feedback/tag";
 
 const meta: Meta = {
 	title: "Components/Tag",
 	component: Tag,
 	argTypes: {
 		children: Utils.arg("text"),
-		color: Utils.arg(colors),
+		color: Utils.arg(Object.keys(Tag.colors)),
```

Comment:
You should not need to have "Object.keys" here. Just pass the whole object. For example in Button:

https://github.com/moaijs/moai/blob/ba2b9830e3cb82d0561e4e42cd5cd86e88866a4c/lib/docs/src/button.stories.tsx#L18-L19

Reply:
If I pass the whole object, in the select of color in all props section, it will show something like "red,_3SCaHx6bYWTV1Fvq2Ps4ji". That's why I use Object.keys to pass only the color and not the value

Reply:
oh I see that was wrong.. I'll look into that later. For now it's good 👍